_includes/header.html

```diff
@@ -2,9 +2,14 @@
 
   <div class="wrapper">
 
-    <a class="site-title" href="{{ site.baseurl }}/">{{ site.title }}</a>
+    <div class="navLogo">
+      <a class="site-title" href="{{ site.baseurl }}/"><img src="/images/smLogo.png" alt=""></a>
+    </div>
+
+    <!-- <a class="site-title" href="{{ site.baseurl }}/">{{ site.title }}</a> -->
```

Comment:
Ok cool. Just leaving stuff in comments in case needed later but I'm thinking now that if I need something we'll just build it.

Sent from my iPhone

> On 8 Feb 2016, at 9:48 PM, Kevin Brown <notifications@github.com> wrote:
> 
> In _includes/header.html:
> 
> > @@ -2,9 +2,14 @@
> >  
> >    <div class="wrapper">
> >  
> > -    <a class="site-title" href="{{ site.baseurl }}/">{{ site.title }}</a>
> > +    <div class="navLogo">
> > +      <a class="site-title" href="{{ site.baseurl }}/"><img src="/images/smLogo.png" alt=""></a>
> > +    </div>
> > +
> > +    <!-- <a class="site-title" href="{{ site.baseurl }}/">{{ site.title }}</a> -->
> In general I'd go ahead and delete this sort of stuff. It'll be in the initial commit if we ever need to go back.
> 
> —
> Reply to this email directly or view it on GitHub.
>